topic_resources/rspec.md

```diff
@@ -0,0 +1,58 @@
+### RSPEC
+[Rspec](rspec.info)
+
+[Additional Tutorial Info](http://code.tutsplus.com/tutorials/ruby-for-newbies-testing-with-rspec--net-21297)
+
+#### Setting Up
+Install Rspec
+``` gem install rspec ```
+
+For each class file that we create, we create one spec file.
+
+Create the ```chair_spec.rb``` test file.
+```
+require 'spec_helper'
+
+describe Chair do
+
+end
+```
+
+Use a ```before``` block to run a specific block of code before every test is run. In this before block example, we will ensure we have an instance of our chair object before we execute these tests.
+
+```
+before :each do
+    @chair = Chair.new "Spinny", "Yellow", :height
+end
+```
+
+Use the describe block to specify actions of a method.
+```
+describe "#new" do
```

Comment:
Should be ```.new``` - this is a class method, not an instance method... or the example ```describe``` block should call an instance method on Chair.